### DISCUSSION TITLE
SD V2 VRAM memory requirements

### DISCUSSION THREAD

**Discussion by dsienra** [smy20011/dreambooth-gui#80] [General]
Hello, I tried to train SD v2 but I get an error out of VRAM memory,

**Comment by smy20011** [comment]
Make sure you close all the program that could consume GPU memeory. Like chrome or others.

It trains fine with 11Gb varm.

**Reply by dsienra** [reply]
Hello, thanks for the response,  I tried many config combinations with no luck.
I'm getting the same error

**Comment by dsienra** [comment]
I'm getting this error
Is trying to allocate 20.00 MiB
RuntimeError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 12.00 GiB total capacity; 11.24 GiB already allocated; 0 bytes free; 11.29 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

--mixed_precision=fp16
--train_batch_size=1
--gradient_accumulation_steps=1
--use_8bit_adam
--resolution=512
--gradient_checkpointing
--train_text_encoder

This is my config, works perfect with sd 1.4 or 1.5 model.
Any idea?

Thanks.
